backend/src/database/User.js

```diff
@@ -17,11 +17,18 @@ const createNewUser = async (newUser) => {
 };
 
 const doesUsernameExist = async (newUser) => {
-  const checkUser = await pool.query(
```

Comment:
Keep this function as-is. These changes don't make any sense. You can avoid all of this if you just pass in the username instead of keeping it an object